### DISCUSSION TITLE
Linux or Windows

### DISCUSSION THREAD

**Discussion by HyakuzaO** [kyleskom/NBA-Machine-Learning-Sports-Betting#405] [Q&A]
Hello guys, Which Linux operating system do you recommend for running the script effectively? I've installed a few versions but then they don't have the necessary compatible libraries, etc. I wanted to install a system that would be compatible with this script without using Windows WSL. Which Linux distribution do you advise? Thank you very much to whoever answers. Regards to all.

**Comment by Scandianis** [comment]
If using the cloud is an option for you, I can recommend (using self 2.5 years) the Oracle Cloud Free Tier (https://www.oracle.com/cloud/free/) and for free use two AMD Compute VMs with Oracle Linux Server installed. Access from windows with Putty software. It is compiled from Red Hat Enterprise Linux (RHEL) source code.

**Reply by HyakuzaO** [reply]
That cloud works well in free version or we must pay? Many regards for the help that you are giving me bro.

**Reply by Scandianis** [reply]
It is totally free. If you in the configuration add more than default processor/memory etc. more cost will apply.

**Reply by HyakuzaO** [reply]
But for run that script we can use in version free?

**Reply by Scandianis** [reply]
Can run for free any software with free licence use. According to ChatGPT:

> What licence paradigm is Python using?
ChatGPT
Python uses the Python Software Foundation License, which is a permissive open-source license. The license is sometimes referred to as the "Python license" or "PSF license." It is a simple and permissive license that allows users to freely use, modify, and distribute Python, including in commercial projects. The license is designed to encourage collaboration and the sharing of code within the Python community. It is an open-source license that aligns with the principles of free software.
